python/smashbox/utilities/__init__.py

```diff
@@ -26,6 +27,32 @@ def setup_test():
     If exception is raised then the testcase execution is aborted and smashbox terminates with non-zero exit code,
 
     """
+    cmd = config.oc_sync_cmd
+    if cmd[0 - len(' --trust'):] == ' --trust':
+        cmd = cmd[:0 - len(' --trust')]
```

Comment:
Please have a look on test_manager pull request. What about creating a class called test_manager, instead of creating just a raw function? It will enable to add much more plugins to the smashbox, just adding another class to the Test_Manager class itself.